### PR TITLE
preserve-books-data

### PR DESCRIPTION
**Pull request by Kgomotso196**
# PROJECT TO DOs
### IN THIS MILESTONE I HAVE:
- Preserved all books data by storing collections in .json files.

### CODE REVIEW

File: DATABASE/books.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[
+  "{\"id\":353,\"title\":\"The Fur Coat\",\"publisher\":\"Troos\",\"cover_state\":\"good\",\"publish_date\":\"2012-12-12\"}",
+  "\"#<Label:0x00000227a3c89138>\""
```
Comment (aradradev) [inline DATABASE/books.json:3]: Optional: @Kgomotso196 Great job implementing the JSON to store and persist data, kinda try to reformat your file to be more readable take that as a suggestion. You could achieve it like so:
```
 @authors << {
      'first_name' => first_name,
      'last_name' => last_name
    }

    @games << {
      'game_name' => game_name,
      'first_name' => first_name,
      'last_name' => last_name,
      'publish_date' => publish_date,
      'multiplayer' => multiplayer,
      'last_played_at' => last_played_at
    }
```

File: classes/book_details.rb
Comment (aradradev) [inline classes/book_details.rb:1]: Good implementation

**Review by aradradev — APPROVED** [review]
Hey @Kgomotso196 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: DATABASE/books.json (hunk 1 above)
Comment (Kgomotso196) [reply DATABASE/books.json:3]: @aradradev Thank you. Working on that up next.

**Pull request merged**